zuul.d/projects.yaml

```diff
@@ -1008,6 +1009,7 @@
     templates:
       - ansible-collections-community-okd
       - publish-to-galaxy-3pci
+      - publish-to-automation-hub
```

Comment:
community.okd is a bit annoying because the repository is in another namespace that we don't control.

So, this one won't work. We need a publish-to-automation-hub-3pci based on https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L1373-L1393.

The existing publish-to-automation-hub templates uses `check` and `gate` pipelines. Both are not in use when the Zuul CI is just a third party CI (3pci).  Instead, we should use the `third-party-check` pipeline, you can take a look at publish-to-galaxy-3pci for an example.